lua/ttt2/libraries/database.lua

```diff
@@ -198,37 +207,47 @@ local function OnChange(index, itemName, key, newValue)
 	dataEntry[key] = newValue
 	storedData[itemName] = dataEntry
 
-	if oldValue == newValue then return end
+	if oldValue == newValue then
+		return
+	end
 
 	-- Get all callbacks
 	local accessName = dataTable.accessName
 	local cache = callbackCache[accessName]
 
-	if not cache then return end
+	if not cache then
+		return
+	end
 
 	-- Call all callbacks
-	local itemNameTable = {itemName, allCallbackString}
+	local itemNameTable = { itemName, allCallbackString }
```

Comment:
Is it really nicer to have spaces between brackets?

Reply:
i don't care too much tbh, but it does look nicer to me.